lib/web/sessions.go

```diff
@@ -862,6 +862,9 @@ func (s *sessionCache) invalidateSession(ctx context.Context, sctx *SessionConte
 	if err := clt.DeleteUserAppSessions(ctx, &proto.DeleteUserAppSessionsRequest{Username: sctx.GetUser()}); err != nil {
 		return trace.Wrap(err)
 	}
+	if err := clt.DeleteUserSAMLIdPSessions(ctx, sctx.GetUser()); err != nil {
+		return trace.Wrap(err)
+	}
```

Comment:
Why do we need to delete all sessions? I'd expect both cookies and sessions to be on per-browser basis, yet here we delete everything, which means logout in one browser will invalidate all other active sessions. This may be a nice feature to have, but I think we should be more selective by default.

Reply:
Hmmm I think you are right. Since this is user logging out from IdP, I thought it would be good to remove all active sessions. But for such feature, looks like it would be better to provide "clear session from all browser" kind of option.

Reply:
@Tener here's the commit which now clears user's current session instead of all SAML session. https://github.com/gravitational/teleport/pull/37619/commits/dd5cd08f5af993fa2c7b4eded05b9de7c9d4a481

I've used SAML session cookie to get the session ID as currently, the `websession` state is not updated with SAML session after SAML auth. I assessed to have the `websession` state updated but looks like that demands some more work outside the scope of this PR.  And I believe, retrieving session ID from cookie value for logout doesn't seem fallible option either. Let me know if you think other way!